src/JHipsterNetSampleApplication/Models/Operation.cs

```diff
@@ -6,13 +7,17 @@
 namespace JHipsterNetSampleApplication.Models {
     [Table("operation")]
     public class Operation {
+        public Operation()
+        {
+            Labels = new JoinListFacade<Label, Operation, OperationLabel>(this, OperationLabels);
```

Comment:
Either your use the inline notation like [Label#new](https://github.com/jhipster/jhipster-net-sample-app-template/pull/24/files#diff-7b250f2e701b3f6910d9eac7f425c4c5R11) or the curly bracket style like here.
Please use a convention.